ds_judgements_public_ui/templates/includes/recent_judgments.html

```diff
@@ -1,25 +1,53 @@
-{% load static i18n %}
-<div class="recent-judgments">
-  <div class="recent-judgments__container">
-    <h2 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h2>
-    <ul class="judgment-listing__list">
-      {% for item in recent_judgments %}
-        <li class="judgment-listing__judgment">
-          <span>
-            <span class="judgment-listing__title">
-              <a href="{% url 'detail' item.uri %}">{{ item.name }}</a>
+{% load static i18n waffle_tags %}
+{% flag v3_homepage %}
+  <div class="recent-judgments">
+    <div class="recent-judgments__container-beta">
+      <h2 id="recently-published-judgments" class="recent-judgments__header-beta">Recently published judgments</h2>
+      <ul class="judgment-listing__list-beta">
+        {% for item in recent_judgments %}
+          <li class="judgment-listing__judgment">
+            <span>
+              <span class="judgment-listing__title">
+                <a href="{% url 'detail' item.uri %}">{{ item.name }}</a>
+              </span>
+              <span class="judgment-listing__court">{{ item.court.name }}</span>
             </span>
-            <span class="judgment-listing__court">{{ item.court.name }}</span>
-          </span>
-          <span>
-            <span class="judgment-listing__neutralcitation">{{ item.neutral_citation }}</span>
-            <time class="judgment-listing__date" datetime="{{ item.date }}">{{ item.date|date:"d M Y" }}</time>
-          </span>
-        </li>
-      {% endfor %}
-    </ul>
-    <p class="judgment-listing__more-info">
-      <a href="{% url 'search' %}">{% translate "judgments.allrecent" %}</a>
-    </p>
+            <span>
+              <span class="judgment-listing__neutralcitation">{{ item.neutral_citation }}</span>
+              <time class="judgment-listing__date" datetime="{{ item.date }}">{{ item.date|date:"d M Y" }}</time>
+            </span>
+          </li>
+        {% endfor %}
+      </ul>
+
+      <p class="recent-judgments__more-info-beta">
+        <a href="{% url 'search' %}">View all judgments</a>
+      </p>
+    </div>
+  </div>
+{% else %}
+  <div class="recent-judgments">
+    <div class="recent-judgments__container">
+      <h2 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h2>
+      <ul class="judgment-listing__list">
+        {% for item in recent_judgments %}
```

Comment:
Just a note for when you rebase this - the change in this file is just that this has been extracted out :)